README.md

```diff
@@ -1,4 +1,22 @@
-## Inventory
+# Inventory
+
+Demo app for showing how Ditto sync and counters work.
+
+This inventory demo showcases the smoothness of the Ditto sync and how counters work with Ditto. You can open up the Presence Viewer to see the Ditto connections too.
```

Comment:
"the smoothness of Ditto's sync and conflict resolution, and how counters work with Ditto. You can also open up the Presence Viewer to see all existing devices and connections in the mesh"